chatbot/db/db.js

```diff
@@ -2,10 +2,15 @@ const STATES = require('../SessionStateEnum.js')
 const SessionState = require('../SessionState.js')
 const Installation = require('../Installation.js')
 const { Pool } = require('pg')
+require('dotenv').config();
```

Comment:
I notice that in our other files we use `.load()` instead of `.config()`. Looking at the `dotenv` docs, it looks like `.config()` has been the preferred command since version 2 and becomes the only command as of version 7 (https://github.com/motdotla/dotenv/blob/HEAD/CHANGELOG.md#700---2019-03-12). We are currently using version 6.1.

I suggest that we take this opportunity to update the other places where `.load()` is used to instead use `.config()`? Or make a new ClickUp task for it and I can do it :-)

Reply:
I feel like despite being super small its out of scope for this so I made [CU-bjq8m1](https://app.clickup.com/t/bjq8m1)